department-of-veterans-affairs/caseflow
# UI for marking users inactive

Resolves #12212

### Description
Creates a "User management" page for team admins to mark users active or inactive

#### Main view
<img width="730" alt="Screen Shot 2019-10-09 at 2 54 47 PM" src="https://user-images.githubusercontent.com/45575454/66511496-0a9bd900-eaa5-11e9-9e85-035b94b494bf.png">

#### Searching for a user
<img width="729" alt="Screen Shot 2019-10-09 at 2 55 02 PM" src="https://user-images.githubusercontent.com/45575454/66511506-12f41400-eaa5-11e9-959a-ab92c5d9bb76.png">

#### Making users inactive
<img width="731" alt="Screen Shot 2019-10-09 at 2 55 08 PM" src="https://user-images.githubusercontent.com/45575454/66511521-16879b00-eaa5-11e9-83c4-ae0fa6d72e22.png">

#### Making users active
<img width="730" alt="Screen Shot 2019-10-09 at 2 55 14 PM" src="https://user-images.githubusercontent.com/45575454/66511526-18515e80-eaa5-11e9-9ba5-8d4ebe801286.png">

#### Search error
<img width="728" alt="Screen Shot 2019-10-09 at 2 55 38 PM" src="https://user-images.githubusercontent.com/45575454/66511536-1e473f80-eaa5-11e9-957a-9b6acc65fc5c.png">

### Acceptance Criteria
- [ ] Admins can mark users in/active in the caseflow application

### Testing Plan
1. Sign in as TEAM_ADMIN
2. Click on the "TEAM_ADMIN (VACO)" dropdown in the navigation bar
3. Ensure there is a link to "Caseflow user management". Click the link
4. Search for a user you want to make inactive (I used BVAKKEELING)
5. Click "Make user inactive"
6. Confirm in the rails console that this user is indeed inactive
```ruby
User.find_by(css_id: "BVAKKEELING").status
# => "inactive"
```
7. Make this user active again by clicking "Make user active"
8. Confirm in the rails console that this user is indeed inactive
```ruby
User.find_by(css_id: "BVAKKEELING").status
# => "active"
9. Search for a nonexistant user (I used BVAIWIZ)
10. Ensure an error pops up
```
